tests/data_context/abstract_data_context/test_store_config_crud.py

```diff
@@ -0,0 +1,49 @@
+from unittest import mock
+
+import pytest
+
+from great_expectations.data_context import EphemeralDataContext
+
+SETTER_METHOD_NAMES = [
+    "expectations_store_name",
+    "validations_store_name",
+    "profiler_store_name",
+    "checkpoint_store_name",
+]
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "store_setter_method_name",
+    SETTER_METHOD_NAMES,
+)
+def test_store_name_setters(
+    store_setter_method_name: str,
+    ephemeral_context_with_defaults: EphemeralDataContext,
+):
+    new_store_name = "new_store_name"
+    setattr(ephemeral_context_with_defaults, store_setter_method_name, new_store_name)
+    assert (
+        getattr(ephemeral_context_with_defaults, store_setter_method_name)
+        == new_store_name
+    )
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "store_setter_method_name",
+    SETTER_METHOD_NAMES,
+)
+def test_store_name_setters_persist(
+    store_setter_method_name: str,
+    ephemeral_context_with_defaults: EphemeralDataContext,
+):
+    new_store_name = "new_store_name"
+    with mock.patch(
+        "great_expectations.data_context.EphemeralDataContext._save_project_config"
+    ) as mock_save_project_config:
```

Comment:
Same note here

Reply:
I'm glad we are getting these tests :rocket: